storage/bootstrap/bootstrapper/commitlog/options.go

```diff
@@ -25,16 +25,25 @@ import (
 	"github.com/m3db/m3db/storage/bootstrap/result"
 )
 
+const (
+	defaultEncodingConcurrency   = 4
+	defaultMergeShardConcurrency = 4
+)
+
 type options struct {
-	resultOpts    result.Options
-	commitLogOpts commitlog.Options
+	resultOpts            result.Options
+	commitLogOpts         commitlog.Options
+	encodingConcurrency   int
+	mergeShardConcurrency int
 }
 
 // NewOptions creates new bootstrap options
 func NewOptions() Options {
 	return &options{
-		resultOpts:    result.NewOptions(),
-		commitLogOpts: commitlog.NewOptions(),
+		resultOpts:            result.NewOptions(),
+		commitLogOpts:         commitlog.NewOptions(),
+		encodingConcurrency:   defaultEncodingConcurrency,
+		mergeShardConcurrency: defaultMergeShardConcurrency,
```

Comment:
maybe check if the `concurrencies are >= 1` in `Validate()`

Reply:
done